solidity/contracts/BondedECDSAKeepFactory.sol

```diff
@@ -239,7 +239,9 @@ contract BondedECDSAKeepFactory is
         uint256 _bond,
         uint256 _stakeLockDuration
     ) external payable returns (address keepAddress) {
+        require(_groupSize > 0, "Minimum signing group size must be greater than 0");
```

Comment:
Maybe `Signing group size must be greater than 0` or `Minimum signing group size is 1`?